ts/tree/tree.ts

```diff
@@ -66,7 +66,7 @@ export class Tree {
         mergeOffsetLower: number,
         subtreeDepth: number
     ): Witness {
-        const mergeSize = 1 << subtreeDepth;
+        const mergeSize = 2 ** subtreeDepth;
```

Comment:
nitpick: Some of the "bitshifts to pow" changes are out of the scope of this PR, we could create a new PR for them. But feel free to ignore this.

Reply:
Will revert and open a separate PR.